### DISCUSSION TITLE
Set transient-display-buffer-action differently for different transients

### DISCUSSION THREAD

**Discussion by bestlem** [magit/transient#303] [Ideas]
Is there a way of making different transients show in different windows.

For example for magit I use transient-posframe as I don't need to see the magic status buffer when choosing actions but don't want to alter my focus so transient appears in centre screen.
However for other transients, especially those which can stay around I would like to show the transient and the buffer it applies to, so not using transient-posframe but say the default transient-display-buffer-action

**Comment by tarsius** [comment]
That isn't currently possible, but I was already planning to look into it eventually. I cannot promise that I'll get to it any time soon though.

**Comment by josephmturner** [comment]
How about https://github.com/magit/transient/pull/332 ?

**Reply by tarsius** [reply]
Merged!
